tests_backend/run_backend_tests.ts

```diff
@@ -24,7 +24,7 @@ let backendType: 'none' | 'local' | 'upstash' | null = null
 const noRedisEnvFilePath = await getEnvFile('none')
 const localRedisEnvFilePath = await getEnvFile('local')
 
-async function getAdminSupabaseTokens(): Promise<{ url: string; serviceKey: string }> {
+async function getAdminSupabaseTokens(): Promise<{ url: string; serviceKey: string; anonToken: string }> {
```

Comment:
we could give a better name than `anonToken` here, something like `anonymousToken`, `publicToken`, etc.; something that auto-explains what this thing is

Reply:
This is called `supa_anon` in config.js `anon key` in `supabase status`. Anon is the term used officially by supabase

Reply:
Thanks for clarifying!